### DISCUSSION TITLE
[v3] Problem with chain $this->redirect()->with()

### DISCUSSION THREAD

**Discussion by qwert1x** [livewire/livewire#6176] [1. Help]
v3.0.0-beta.6
We can't use chain 
`return $this->redirect('/posts')->with('status', 'Post updated!');` 
according https://livewire.laravel.com/docs/redirecting#flash-messages because `$this->redirect()` is void (?) and we get "Call to a member function with() on null" error.
Need to update docs or method.
![2023-08-03_18-08-25](https://github.com/livewire/livewire/assets/471334/90c087db-91d0-4a4e-b617-d370ce26fa0d)

**Comment by joshhanley** [comment]
@qwert1x thanks for reporting! Any chance you could submit a failing test PR? Thanks!

**Comment by wiyatnanto** [comment]
try to use session()->flash('success', 'Post updated!'); if we use navigation: true

**Comment by skeemer** [comment]
With how `\Livewire\Features\SupportRedirects\HandlesRedirects` is currently coded, there is no way to do what is shown in the docs. Should the docs be changed? Something like:
```
$this->skipRender();
return redirect('/posts')->with('status', 'Post updated!');
```
I'm assuming that the `$this->` was a typo as the `$this->redirect*()` functions are and have been void. But I'm not sure if it automatically adds a skipRender if you return a redirect from an action or not.
